src/ui/components/AddBounty.tsx

```diff
@@ -0,0 +1,161 @@
+import React, {useMemo} from 'react';
+import {StyleSheet, View} from 'react-native';
+import {TextInput, Button, HelperText, Subheading} from '@ui/library';
+import {Layout} from '@ui/components/Layout';
+import globalStyles, {standardPadding} from '@ui/styles';
+import {Account} from 'src/api/hooks/useAccount';
+import {SelectAccount} from '@ui/components/SelectAccount';
+import {Padder} from '@ui/components/Padder';
+import {useApiTx} from 'src/api/hooks/useApiTx';
+import {useApi} from 'context/ChainApiContext';
+import {useSnackbar} from 'context/SnackbarContext';
+import {InputLabel} from '@ui/library/InputLabel';
+import {decimalKeypad} from 'src/utils';
+import {useFormatBalance} from 'src/hooks/useFormatBalance';
+import {useBountiesSummary} from 'src/api/hooks/useBountiesSummary';
+import type {BN} from '@polkadot/util';
+import {formattedStringToBn} from 'src/utils/balance';
+import {countUtf8Bytes} from 'src/utils';
+
+type Props = {
+  onClose: () => void;
+};
+
+export function AddBounty({onClose}: Props) {
```

Comment:
Is it the same UI? Can you add some screenshots for the screens that are different (if any)